src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/AscInsight.cs

```diff
@@ -0,0 +1,103 @@
+        /// It should only expand on any relevant information about the insight.</param>
+        /// <param name="recommendedAction">Recommended action that is specifically addressing what steps thesupport engineer should take.
+        /// If you only want to have the support engineer to send the customer ready content, then that is all you would have to include here.</param>
+        /// <param name="customerReadyContent">This is a response that is meant for the support engineer to paste directly into an email to the customer. 
+        /// It is meant to completely solve the problem, and if that is not the case you may not want to use this field, by passing a null value. 
+        /// Other good information to have is additional resources for the customer so that they can solve the problem in the future.</param>
+        /// <param name="context">Operation context which is passed into detector run method</param>
+        /// <param name="extraNameValuePairs">Additional name value pairs that you want to display in Applens/App Service Diagnostics. These will not be added in Azure Support Center.</param>
+        /// <param name="isExpanded">Whether you want to Applens/App Service Diagnostics to expand the insight initially</param>
+        /// <returns>Azure Support Center Insight Object</returns>
+        /// <example> 
+        /// This sample shows how to use the <see cref="AddAscInsight"/> method.
+        /// <code>
+        /// public async static Task<![CDATA[<Response>]]> Run(DataProviders dp, OperationContext cxt, Response res)
+        /// {
+        ///     res.AddAscInsight(
+        ///         "Failed Scale Operation Detected", // Do not put resource specific information here
+        ///         InsightStatus.Critical,
+        ///         new Text(@"###A scale operation failed because there was already a scale operation underway. Here is the error:
+        ///                    
+        ///                    ```
+        ///                    Operation failed because a current scale operation is ongoing.
+        ///                    ```
+        ///         ", true), // This is markdown
+        ///         new Text(@"Copy and paste the Customer Ready Content and send to the customer"), // This is not markdown
+        ///         new Text(@$"Your scale operation for site ***{cxt.Resource.Name}*** failed because there was a current scale operation underway. 
+        ///                    
+        ///                    Please wait for the current operation to finish and then try again to scale your app service plan. ", true),
+        ///         cxt,
+        ///         new Dictionary<string, string>() {
+        ///             { "Header", "<markdown>This is extra insight members that you want to be shown in ***Applens*** and ***App Service Diagnostics***.</markdown>"
```

Comment:
I am guessing this will break document generation. you would need to use CDATA to preserve html tags.
Just test this out locally:
1. Install docfx : https://github.com/dotnet/docfx/releases (save the folder and update PATH)
2. Go to 'data\documentation' folder in our project root
3. run 'docfx'  **(this is the step that needs to pass)**
4. run 'docfx serve _site'

Reply:
Perfect I was going to ask you how to do this. I will figure out how to make it work.

Reply:
I have fixed this by removing that dictionary section with the markdown tags. Not the example looks like this, though the formatting is off because of the strange markdown strings, not sure what to do about that. 

![image](https://user-images.githubusercontent.com/7739017/44800155-91744780-ab6a-11e8-9948-73d786414391.png)

Reply:
Also do I check in the .manifest file?

Reply:
ya, you can add that too.